pages/linux/flameshot.md

```diff
@@ -4,30 +4,34 @@
 > Supports basic image editing, such as text, shapes, colors, and imgur.
 > More information: <https://flameshot.org>.
 
-- Launch flameshot with a simpler interactive mode:
+- Create a fullscreen screenshot:
 
-`flameshot launcher`
+`flameshot full`
 
-- Launch flameshot and immediately start interactively annotating parts of the screen to screenshot:
+- Create a screenshot interactively:
 
 `flameshot gui`
 
-- Take a full screenshot (all monitors):
+- Create a screenshot and save it to a specific path:
 
-`flameshot full`
+`flameshot gui --path {{path/to/directory}}`
+
+- Create a screenshot interactively in a simplified mode:
 
-- Take a screenshot from monitor 1:
+`flameshot launcher`
+
+- Create a screenshot from a specific monitor:
 
-`flameshot screen --number {{1}}`
+`flameshot screen --number {{5}}`
 
-- Set the save path to write screenshots to:
+- Create a screenshot and print it to the standard output:
 
-`flameshot full --path {{path/to/directory}}`
+`flameshot gui --raw`
 
-- Delay the screenshot for N milliseconds and output to clipboard:
+- Create a screenshot and copy it to a clipboard:
```

Comment:
```suggestion
- Create a screenshot and copy it to the clipboard:
```

Reply:
I think it's better to use `a` here: we are not talking about a specific clipboard.

Reply:
I thought there are [several][wiki] types of clipboard in linux. 🤔 

[wiki]: https://en.wikipedia.org/wiki/Clipboard_(computing)#:~:text=The%20X%20Window%20System%20commonly,various%20selections%20is%20not%20standardized.